### ISSUE TITLE
[DOCS] Add example to Alert Actions documentation showing use of mustache "current element"

### ISSUE THREAD

**Issue by peluja1012** [elastic/security-docs#453]
## Description 

We would like to add an example to the [Alert Notification Placeholder](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-action-variables) section of the docs showing the use of the `Mustache` "current element" notation for array fields. It would be beneficial to document something like the statement below, which will output all of the rule references in the `signal.rule.references` array:
```
{{#signal.rule.references}} {{.}} {{/signal.rule.references}}
```
We could add it as another example like the one already shown in that section of the docs:
![image](https://user-images.githubusercontent.com/616158/105030910-7ef05680-5a22-11eb-97c9-1afc93816340.png)

**Comment by jmikell821** [comment]
Hey @peluja1012 I think I'll add a subsection to the [Alert Placeholders topic (preview)](https://security-docs_450.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#rule-action-variables) and include this additional example. 

Currently, in the preview, I have this example, per the PR: 
> An array contains each alert generated since the last time the action was executed. For example, ``{{#context.alerts}}Detection alert for user: {{user.name}}{{/context.alerts}}` will create the string Detection alert for user: <user.name> for every alert in the array.

But will use yours. Let me know if I should add any more examples while I currently have "Creating detection rules" open in PR #450.

**Comment by jmikell821** [comment]
@peluja1012 this is PR #450. I created a new subsection, which you can preview [here](https://security-docs_450.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#placeholder-examples). Can you please double check the syntax of this and the "Alert Placeholders" section above to ensure it's 100% accurate? Thanks!

**Comment by jmikell821** [comment]
Merged #450.
